tests/ci/run_accp_test_integration.sh

```diff
@@ -25,7 +25,7 @@ version=$($TEST_JAVA_HOME/bin/java -version 2>&1 | head -1 | cut -d'"' -f2 | sed
 # The JDK version should be least 10 for a regular ACCP build. We can
 # still test on older versions with the TEST_JAVA_HOME property.
 if (( "$version" <= "10" )); then
-	./gradlew -DTEST_JAVA_HOME=$TEST_JAVA_HOME -DFIPS=$testing_fips test_integration
+	./gradlew -DTEST_JAVA_HOME=$TEST_JAVA_HOME -DTEST_JAVA_MAJOR_VERSION=$version -DFIPS=$testing_fips test_integration
```

Comment:
why is this now necessary? doesn't -DTEST_JAVA_HOME tell which JAVA_HOME to use for running test, and thus which version to use?

Reply:
We don't (currently) extract the major version information from the TEST_JAVA_HOME path and need that value to gate some logic in cmake. So, we need to pass it in explicitly. (Technically, it should already have been on this command-line, but previously we only cared if the version was sufficiently old, not if it was sufficiently new.

Reply:
i see, that makes sense. thanks.